### PR TITLE
Add code coverage report

### PR DESCRIPTION
**Pull request by mikker**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ gem 'rubocop', require: nil
 gem 'timecop'
 gem 'webmock'
 
+gem 'codecov', require: false, group: :test
```
Comment (kuisathaverat) [inline Gemfile]:
```suggestion
gem 'codecov', require: false, group: :test
gem 'simplecov-cobertura', require: false, group: :test
```

**Comment by mikker** [comment]
How do I check for Jenkins? `ENV['CI']` isn't set AFAIK, is it `ENV['JENKINS_FILE']`?

**Comment by kuisathaverat** [comment]
The most common way it is to check BUILD_NUMBER or JOB_NAME environment variables are defined

**Comment by mikker** [comment]
jenkins, retest this please

**Pull request merged**